fava/core/file.py

```diff
@@ -123,9 +123,20 @@ def render_entries(self, entries):
             The entries rendered in Beancount format.
 
         """
+        excl_flags = [
+            flags.FLAG_PADDING,      # P
```

Comment:
If the generated padding transactions are not exported, maybe we should export the Padding directives themselves too in addition to transactions and balances?

Reply:
Would that mean that for each account used in a balance directive in the export, we add a pad directive, like `2000-05-28 pad Assets:CA:BofA:Checking Equity:Opening-Balances`?

Reply:
Mhh not sure actually - we should probably do that if we want the exported file to be a self-contained Beancount file without errors. Currently the balance directive in the exported file are a bit awkward since they probably won't balance on their own with some subset of the entries. Also, re could export pad directives if they already exist in the source.

Reply:
True the file would not be "self-contained" regardless, as the user can select any filters and therefore display a subset of the Journal that cannot possibly be "self-contained".

My use-case here is that I export a subset of my journal (Liabilities) and wrote an Importer for my girlfriend's fava, so she can import "my liabilities", and vice versa, as we have two separate journals.

So I guess we cannot implement this sensibly, and keep it the way it works now, until a use case turns up that would need this strict "self-containment".